cmd/utils/flags.go

```diff
@@ -846,6 +846,11 @@ var (
 		Value:  "0x0000000000000000000000000000000000000000",
 		EnvVar: "ROLLUP_ADDRESS_MANAGER_OWNER_ADDRESS",
 	}
+	RollupDiffDbFlag = cli.Uint64Flag{
+		Name:   "rollup.diffdbcache",
```

Comment:
Setting a sane default here is probably a good idea, otherwise it'll default to 0. Can add a `Value` to the `cli.Uint64Flag`

Reply:
I see you set it to `256` here: https://github.com/ethereum-optimism/go-ethereum/pull/128/files#diff-cb10e99654aa32525bbb358ea722f79e26da92b3ecdc70e29ce1d711f1be3de4R74

It still needs to be imported and set as the `Value` here